code/modules/recycling/sortingmachinery.dm

```diff
@@ -177,7 +177,9 @@
 		var/obj/structure/closet/crate/O = target
 		if(O.opened)
 			return
-		if(use(3))
+		if(amount >= 3 && do_after_once(usr, 15, target = target))

```

Comment:
You know what I did to the last person who cast `usr` when a proc can explicitly pass `user`.

💣 💥

Reply:
Max capped their ass off the codebase

Reply:
hawt

Reply:
TBF, relative to the rest of this proc, that's just a small drop in the ocean :smile: